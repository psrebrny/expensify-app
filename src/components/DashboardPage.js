import React from 'react';
import { ExpenseList } from '../containers/ExpenseList';
import { ExpenseListFilters } from '../containers/ExpenseListFilters';

export const ExpenseDashboardPage = () => {
  return (
      <div>
        <ExpenseListFilters/>
        <ExpenseList/>
      </div>
  );
};