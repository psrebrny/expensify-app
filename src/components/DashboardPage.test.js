import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseDashboardPage } from './DashboardPage';

describe('components/dashboard.js', () => {
  test('render dashboard', () => {
    const wrapper = shallow(<ExpenseDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
  });
});