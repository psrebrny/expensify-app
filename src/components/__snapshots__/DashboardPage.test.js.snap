// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/dashboard.js render dashboard 1`] = `
<div>
  <Connect(ExpenseListFiltersClass) />
  <Connect(ExpenseListClass) />
</div>
`;
